common/src/main/scala/explore/AppMain.scala

```diff
@@ -101,11 +105,30 @@ trait AppMain extends IOApp {
       IO.fromFuture(httpCall).map(_.body)
     }
 
+    val fetchToken: IO[String] = IO("...")
+
+    val connectParameters: IO[JsonObject]                          =
+      fetchToken.map(token => JsonObject("Authorization" -> s"Bearer $token".asJson))
+
+    val reconnectionStrategy: ReconnectionStrategy[IO, CloseEvent] =
+      ReconnectionStrategy(maxAttempts = 100,
```

Comment:
Any reason for a 100? Or is it reset once you reconnect?

Reply:
The attempt counter is indeed reset upon a successful reconnection.

I just put a high number there for now. We have to decide what reconnection strategy we will use, we probably want an exponential backoff instead of a linear one.

How does it sound to have a capped exponential backoff with infinite attempts?

Reply:
Sure that sounds like what the seqexec does
Would you do that in a separate PR?

Reply:
Didn't think of copying what we do in Seqexec. I'm sending a new commit with changes for clue 0.6.0 and I'm including the same logic as there.